index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div id="status-bar" class="flex">
+      <div id="time">9:41</div>
+      <div class="flex" id="bars">
+        <img src="img/Cellular-Connection.png" alt="">
+        <img src="img/Wifi.png" alt="">
+        <img src="img/Battery.png" alt="">
+      </div>
+    </div>
```

Comment:
- Hey, great job with fixing the changes as suggested by the previous reviewer, however, you've added a part of the design which you're not meant to implement as this part was a screenshot taken from a mobile device. Please, kindly remove it.

![image](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/59397159/c9fadb3f-868b-40b8-8db1-2bc45beef173)